controllers/errors.controllers.js

```diff
@@ -10,10 +8,15 @@ exports.handleCustomErrors = (err, req, res, next) => {
 exports.handlePSQLerrors = (err, req, res, next) => {
   if (err.code === "22P02") {
     res.status(400).send({ msg: "bad request" });
+  } else if (err.code === "23502") {
```

Comment:
As this if-statement grows, it would be worth thinking about different, more efficient ways of achieving the same thing (e.g a switch statement) - you don't have to do it now (unless you want to!) but it's something to bear in mind!